code/modules/martial_arts/cqc.dm

```diff
@@ -31,7 +31,7 @@
 	var/damage_multiplier = 1 + A.getStaminaLoss() / 100 //The chokehold is more effective the more tired the target is.
 	while(do_mob(A, D, 2 SECONDS) && chokehold_active)
 		D.apply_damage(10 * damage_multiplier, OXY)
-		D.SetLoseBreath(3 SECONDS)
+		D.AdjustLoseBreath(3 SECONDS)
```

Comment:
Mirroring from the discord.
Doing adjustlosebreath is a really bad idea. This technically buffs CQC. What you want to do is do `LoseBreath()` (or wjatever the fuck i called it I'm in bed rn) so that it will increase the value to 3 seconds but it won't decrease it to 3 seconds.

Reply:
done